microlith/entrypoints/loki.sh

```diff
@@ -34,4 +34,4 @@ cat > "${PROMETHEUS_DYNAMIC_INTERNAL_DIR}"/loki.json << __EOF__
 ]
 __EOF__
 
-/usr/bin/loki -config.file="${LOKI_CONFIG_FILE}"
+/usr/bin/loki -config.file="${LOKI_CONFIG_FILE}" -print-config-stderr
```

Comment:
Was this temporary or is this meant to stay?

Reply:
I was thinking it might be useful, there's so much spewing out anyway...

Debating whether to forward to files by default (just need to enable `ENABLE_LOG_TO_FILE`) but that feels wrong as it's hard to see when things fail and more difficult when plumbing into automation too. Arguably we should just provide it all and leave it to the consumer to approach however they want, this is the specific problem we had to solve with server previously with no choice available.

Reply:
IMHO the amount of spew on startup is an issue we'll have to address, so adding more to it won't help that :p

This seems like something that'll be useful, but Loki config files tend to be big (especially pre-2.4) so I'm hesitant on having this be the default. If you think it's useful I'm happy to leave it though.

Reply:
I'll drop it for now, I need to have a look at the split logging config - it would be good to log most stuff to file but get warn/errors out on the console which should be doable by any decent log framework.